.gitattributes

```diff
@@ -1,4 +1,6 @@
-*.pbxproj -diff linguist-generated
+*.pbxproj binary linguist-generated
```

Comment:
Does making `pbxproj` a binary also hide it from the diff? Is it still possible to expand the diff to see the changes?

IMHO we would like to hide it from the diff 👍, but I would also like to see the changes if I want to. What do you think?

Reply:
@sjchmiela, yeap, marking file as `binary` disables `diff` and `CRLF` conversion, but from what I've experienced only in `git` tool. That means neither `git show` nor `git diff` would show file content. I believed that this would cause `github` to show this kind of files (with `binary` attribute) to be shown as `binary file` in PRs and previews as well, but it turned out that `github` is not working like this 🤔 So, if you navigate to https://github.com/expo/expo/pull/8508/files you'll see that all `.pbxproj` files contents are perfectly visible if you expand the diff 🤔 Maybe I'm missing sth somewhere, but I dunno where and what 🤷